src/services/search/search.service.js

```diff
@@ -14,7 +14,8 @@ module.exports = function () {
   };
 
   const client = elasticsearch.Client({
-    hosts: 'elastic:changeme@elasticsearch:9200'
+    hosts: 'elastic:changeme@elasticsearch:9200',
```

Comment:
even if its only available in the local secure network we should add pw protection and handle auth via ENV vars